### ISSUE TITLE
Cannot Sign Tags

### ISSUE THREAD

**Issue by rawkode** [pulumiverse/infra#30]
With pulumi-doppler, I ran into an issue where the `release.yaml` workflow was unable to run and it's because I was signing my Git tags.

We should update `release.yaml` to work with these, or add some documentation that tags can't be signed.

**Comment by Frassle** [comment]
This is a limitation of pulumictl, so moving to that repo to track.

**Comment by Frassle** [comment]
https://github.com/pulumi/pulumictl/issues/52

**Comment by Frassle** [comment]
This is now fixed in pulumictl, will be in the next pulumictl release.

**Comment by ringods** [comment]
@Frassle is the `pulumictl` change already in a release?

**Comment by Frassle** [comment]
No, it's having issues publishing because of windows arm builds. Paul was having a look, but I think got enough other stuff on he hasn't got it fixes yet, could maybe nudge Anton as his on op rotation.
